lib/DDG/Rewrite.pm

```diff
@@ -156,8 +156,11 @@ sub _build_nginx_conf {
 
 	$cfg .= "\techo_before_body '".$self->callback."(';\n" if $wrap_jsonp_callback;
 	$cfg .= "\techo_before_body '".$self->callback.qq|("';\n| if $wrap_string_callback;
+	my ($spice_name) = $self->path =~ /js\/spice\/(.+?)\//;
```

Comment:
It will if the spice child directories allow for different upstream endpoints. I think they do right? In the case of dictionary {definition, audio, etc.}?